src/molecules/checkbox-button/checkbox-button.tsx

```diff
@@ -1,8 +1,8 @@
-import { InputTypes } from "../constants/input-types";
+import { InputTypes } from "../../atoms/constants/input-types";
 import React from "react";
-import { Icon } from "../icons/icon";
-import { Icons } from "../constants/icons";
-import "./checkbox.scss";
+import { Icon } from "../../atoms/icons/icon";
+import { Icons } from "../../atoms/constants/icons";
+import "../checkbox-input/checkbox-input.scss"; // Todo: Remove dependency on external styles
```

Comment:
Is there an issue for this? If so, please list it here

Reply:
#86 - All files related to a component should live in its folder. Components that are dependent on another component's styles can create undesired and unexpected problems when stuff gets updated. A better solution would be to have each component's styles self-contained within it's own style sheet and if there are any shared styles, those things should be broken out into a mixin, function, etc.

I removed those comments for now (I hadn't intended them to get committed anyway, sorry!) and we can pick this up in a different PR.